frontend/app/services/AuthenticationService.scala

```diff
@@ -1,9 +1,17 @@
 package services
 
+import com.gu.identity.play.{AccessCredentials, AuthenticatedIdUser}
 import configuration.Config
+import play.api.mvc.RequestHeader
 
 object AuthenticationService extends com.gu.identity.play.AuthenticationService {
   def idWebAppSigninUrl(returnUrl: String) = Config.idWebAppSigninUrl(returnUrl)
 
   val identityKeys = Config.idKeys
+
+  override lazy val authenticatedIdUserProvider: (RequestHeader) => Option[AuthenticatedIdUser] = AuthenticatedIdUser.provider(
+    AccessCredentials.Cookies.authProvider(identityKeys),
+    AccessCredentials.Token.authProvider(identityKeys, "members-data-api")
```

Comment:
Gah, we do want the Membership-Frontend and the Members-Data-API (on members-data-api.theguardian.com) to have the same target-client-id.

I know we spoke about this on Friday, but I now feeling guilty about deciding on `members-data-api` as the common target-client-id, when it should probably be `membership`. No one is actually using it yet (although the mobile team are just getting started with using it!) so maybe we still are in time to change it before all the mobile clients go out! Would you be amenable to that @DiegoVazquezNanini ? If so, this would be the only other code we need to change:

https://github.com/guardian/membership-attribute-service/blob/34406d9a3/membership-attribute-service/app/services/IdentityAuthService.scala#L15